src/metadata.py

```diff
@@ -56,7 +56,7 @@ def get(path: Path) -> types_.Metadata:
         raise InputError(f"Invalid value for name key: {name}, expected a string value")
 
     docs = metadata.get(METADATA_DOCS_KEY)
-    if not isinstance(docs, str | None) or (METADATA_DOCS_KEY in metadata and docs is None):
+    if docs and not isinstance(docs, str) or (METADATA_DOCS_KEY in metadata and docs is None):
```

Comment:
As noted on the other PR, This change leads to to rejecting empty `docs` values which is fine. Should the error message also be updated?